3rdparty/3rdparty.pro

```diff
@@ -2,7 +2,7 @@ TEMPLATE = subdirs
 CONFIG += ordered qt thread
 
 
-SUBDIRS += libneural libcdfread  libcsvparser pugixml/src  libpillow libpls
+SUBDIRS += libneural libcdfread  libcsvparser pugixml/src  libpillow libpls ErrorHandling
```

Comment:
@rish9511 Let's move ErrorHandling library out of 3rd party? Can you add this to the TODO?

Reply:
@sahil21 
sure